mitosheet/mitosheet/tests/test_parse.py

```diff
@@ -1134,6 +1134,41 @@ def test_parse(formula, column_header, formula_label, df, python_code, functions
         )
 
 
+# Right now, VLOOKUP is the only formula that allows cross-sheet referencing. In the future, 
+# other cross-sheet references in parser can be added here. 
+VLOOKUP_TESTS = [
+    (
+        '=VLOOKUP(A0, df_2!A:B, 2)',
+        'B',
+        0,
+        [
+            pd.DataFrame(
```

Comment:
Let's change the basic test to make sure the two dataframes don't share any columns - else we'll run into the bug we hit above.